package/yast2-bootloader.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Jan 16 16:51:54 UTC 2018 - jreidinger@suse.com
+
+- fix crash for getting partition to activate when stage1 device
+  is disk (bsc#10761210)
```

Comment:
Bug #10761210 does not exist, according to bugzilla.suse.com. Wrong copy&paste?

Reply:
Yes, you wrote an extra 0 at the end.

Reply:
ah, typo. I use 0 instead of ) and appending ) does not fix it :)